hp-ux/generate_wazuh_packages.sh

```diff
@@ -32,9 +34,16 @@ build_environment() {
   fi
 
   echo "Installing dependencies."
+  depot=""
+  if [ -z "$depot_path" ]
```

Comment:
This variable should be enclosed by `{` and `}`. This will avoid any possible issues related to the variable interpretation and improves readability.

Reply:
Done in 51e3cffa812d7f2ec8ee20dd57029219c3751586